src/coreclr/jit/lsra.h

```diff
@@ -486,6 +486,7 @@ class RegRecord : public Referenceable
         previousInterval = nullptr;
         regNum           = REG_NA;
         isCalleeSave     = false;
+        regOrder         = UCHAR_MAX;
```

Comment:
Why this change?

Reply:
If registers are not included in `REG_VAR_ORDER_FLT` and `REG_VAR_ORDER`, `regOrder` value for the registers are not set and can have a garbage value. Then LSRA can allocate differently each time and the test will fail. This code sets to lowest order for registers not included `REG_VAR_ORDER_*`.
I thought it is better to have the lowest order value than garbage value for any mistakes (missing regs in any other future archs).
If you think it is must to set all regs in `REG_VAR_ORDER_*`, I will remove this. Thank you.

Reply:
Ok, let's keep it.. I just noticed that it showed up as a (very minor) throughput regression.